sys/include/net/netstats.h

```diff
@@ -53,6 +65,27 @@ typedef struct {
     uint32_t rx_bytes;          /**< received bytes */
 } netstats_t;
 
+/**
+ * @brief       Stats per peer struct
+ */
+typedef struct netstats_nb {
+    uint8_t l2_addr[8];     /**< Link layer address of the neighbor */
+    uint8_t l2_addr_len;    /**< Length of netstats_nb::l2_addr */
+    uint16_t etx;           /**< ETX of this peer */
+#ifdef MODULE_NETSTATS_NEIGHBOR_EXT
+    uint8_t rssi;           /**< Average RSSI of received frames in abs([dBm]) */
+    uint8_t lqi;            /**< Average LQI of received frames */
+    uint32_t tx_count;      /**< Number of sent frames to this peer */
+    uint32_t tx_failed;     /**< Number of failed transmission tries to this peer */
+    uint32_t rx_count;      /**< Number of received frames */
+    uint32_t tx_bytes;      /**< Bytes sent */
+    uint32_t rx_bytes;      /**< Bytes received */
```

Comment:
If those are just for printing out information, I'd rather drop them - 20 bytes / neighbor is quite a lot to keep around for curiosity's sake.